_posts/blog/2022-12-1-vector-library-vs-vector-database.md

```diff
@@ -15,16 +15,16 @@ In the world of Vector Search, we use **vector embeddings** – generated by **M
 
 We can use [vector distance functions](https://weaviate.io/blog/2022/09/Distance-Metrics-in-Vector-Search.html){:target="_blank"} like euclidean distance or cosine distance to determine if these objects are similar to other objects in the collection. However, to do this we need to compare the distances between the query vector and **every** vector in the collection. This type of calculation can be difficult to scale to millions or billions of vectors.
 
-This is why we have **Vector Databases** (aka Vector Search Engines) and **Vector Libraries**. They both use the Approximate Nearest Neighbor (ANN) algorithm to search through vectors in a tiny fraction of the time. You can learn more about this topic from ["Why Vectors Search is so Fast."](https://weaviate.io/blog/2022/09/Why-is-Vector-Search-so-fast.html){:target="_blank"}
+This is why we have **Vector Databases** (aka Vector Search Engines) and **Vector Libraries**. They both use Approximate Nearest Neighbor (ANN) algorithms to search through vectors in a tiny fraction of the time. You can learn more about this topic at ["Why is Vector Search so fast?"](https://weaviate.io/blog/2022/09/Why-is-Vector-Search-so-fast.html){:target="_blank"}.
 
 ## The Big Question
-So, if both vector databases and vector libraries allow you to efficiently search through your vectors. What are the key differences between them, and why/when should you choose one over the other?
+So, both vector databases and vector libraries allow you to efficiently search through your vectors. What are the key differences between them, and why/when should you choose one over the other?
 
 ## Vector Libraries
-Vector libraries store vector embeddings in in-memory indexes, in order to perform similarity search. Most vector libraries share the following characteristics:
+Vector libraries store vector embeddings in in-memory indexes, in order to perform similarity searches. Most vector libraries share the following characteristics:
 1. they store vectors only,
 2. index data is immutable,
-3. query during import limitation 
+3. querying during import is limited
```

Comment:
I think the original version is fine. In libraries you can't query during import at all, so "is limited" isn't accurate.

Reply:
My suggestion was about using consistent language. The first two items are in the active voice format ("store vectors", "is immutable"), while the 3rd had no verb at all. "query limitation" and "query is limited" mean the same to me (they're both vague - what is the limitation exactly? It could be that only the query could be queued, or that only some records would be returned etc.), yet "is limited" is in the active voice format.

In light of your comment, I'll revise my suggestion to "3. querying during import is impossible". What do you think?